quickwit-search/src/collector.rs

```diff
@@ -321,7 +321,8 @@ fn merge_leaf_responses(
         .sum();
     let failed_splits = leaf_responses
         .iter()
-        .flat_map(|leaf_response| leaf_response.failed_splits.iter().cloned())
+        .flat_map(|leaf_response| leaf_response.failed_splits.iter())
```

Comment:
It really does not matter does it?

Reply:
Yes, it just make code easier to extend in the future in case: in the future, we want to perform post-action right after flat_map